doc/man3/SSL_CONF_cmd.pod

```diff
@@ -85,6 +85,15 @@ its preference list. This usually indicates a client without AES hardware
 acceleration (e.g. mobile) is in use. Equivalent to B<SSL_OP_PRIORITIZE_CHACHA>.
 Only used by servers. Requires B<-serverpref>.
 
+=item B<-legacy_ssl_eof>
+
+Some TLS implementations do not send close_notify on shutdown. Before OpenSSL
+3.0 the SSL_ERROR_SYSCALL with errno=0 was reported in such cases. This
```

Comment:
Too many "the"s:
Before OpenSSL 3.0, SSL_ERROR_SYSCALL

Reply:
Fixed. Thanks!